Common/Sources/Extensions/UIViewController.swift

```diff
@@ -63,7 +63,7 @@ extension UIViewController: HyperwalletFlowDelegate {
         static var coordinator =  [ObjectIdentifier: HyperwalletCoordinator]()
         static var initializationData = [ObjectIdentifier: [InitializationDataField: Any]]()
     }
-    public weak var flowDelegate: HyperwalletFlowDelegate? {
+    public var flowDelegate: HyperwalletFlowDelegate? {
```

Comment:
revert it and disable swiftlint rule deleting this.

Reply:
fixed